src/Provider/DatabaseMenuProvider.php

```diff
@@ -44,7 +45,7 @@ public function __construct(BuilderInterface $builder, RequestStack $requestStac
      * @return \Knp\Menu\ItemInterface
      * @throws \InvalidArgumentException if the menu does not exists
      */
-    public function get($name, array $options = array())
+    public function get(string $name, array $options = []): ItemInterface
```

Comment:
Is dit niet ook het moment om dan de @param en @return in de comments te verwijderen? (non blocking comment)